src/main/scala/lotto/domain/LottoNumber.scala

```diff
@@ -0,0 +1,32 @@
+package lotto.domain
+
+import lotto.domain.LottoNumber.isValidRangeNumber
+
+
+object LottoNumber {
+    private val MIN_LOTTO_NUMBER = 1
+    private val MAX_LOTTO_NUMBER = 45
+    val CACHED_LOTTO_NUMBER: Map[Int, LottoNumber] = {
+        (LottoNumber.MIN_LOTTO_NUMBER to MAX_LOTTO_NUMBER)
+          .map { num => num -> new LottoNumber(num) }
+          .toMap
+    }
+
+    def apply(number: Int): LottoNumber = {
+        require(MIN_LOTTO_NUMBER <= number && number <= MAX_LOTTO_NUMBER)
```

Comment:
isValidRangeNumber로 사용할 수 있지않을까요?

```suggestion
        require(isValidRangeNumber(number))
```